scripts/circleci/release/installScalyrAgentV2.sh

```diff
@@ -158,6 +166,9 @@ while (( $# > 0)); do
     --extract-packages)
       echo "Extracting...";
       untar_tarball $0 || die "Failed to extract packages";
+      pwd
```

Comment:
Remember to remove this and the next few lines.  Looks like debugging output.

Reply:
Oops, thanks.